nat-lab/tests/utils/router/linux_router.py

```diff
@@ -429,6 +429,35 @@ async def break_udp_conn_to_host(self, address: str) -> AsyncIterator:
                 "icmp-host-unreachable",
             ]).execute()
 
+    @asynccontextmanager
+    async def block_udp_port(self, port: int) -> AsyncIterator:
```

Comment:
Add documentation for this method, explaining that this does.
In future it will be very unclear, are we blocking incoming outgoing, or what kind of port, and why is this needed